testing/unit_testing/insert_cohort_test/test_InsertCohort.pf

```diff
@@ -0,0 +1,331 @@
+module test_InsertCohort
```

Comment:
This test module provides a very nice set of tests of InsertCohort - this is a really valuable addition!